akka-cluster-tools/src/main/resources/reference.conf

```diff
@@ -75,3 +93,44 @@ akka.cluster.client {
 # //#cluster-client-mailbox-config
 
 
+akka.cluster.singleton.default {
```

Comment:
You can have many singletons, and this only defines the settings that are used if you create those settings via `ClusterSingletonManagerSettings(system)`, but I think you might be right that it will make sense anyway without the `.default`. I'll take a look at that.